fe2-android/app/src/main/java/com/github/dedis/popstellar/repository/remote/LAONetworkManager.java

```diff
@@ -81,7 +84,13 @@ private void resubscribeToChannelOnReconnection() {
             .subscribe(
                 event ->
                     subscribedChannels.forEach(
-                        channel -> disposables.add(subscribe(channel).subscribe()))));
+                        channel ->
+                            disposables.add(
+                                subscribe(channel)
+                                    .subscribe(
+                                        () -> Log.d(TAG, "nothing to see"),
+                                        error -> Log.d(TAG, "error here too")))),
```

Comment:
should we maybe put some more explicit errors?

Reply:
Might help indeed :)